bentoml/_internal/models/model.py

```diff
@@ -56,19 +58,25 @@ class ModelSignatureDict(t.TypedDict, total=False):
 CUSTOM_OBJECTS_FILENAME = "custom_objects.pkl"
 
 
-@attr.define
 class ModelOptions:
+    def with_options(self, **kwargs: t.Any) -> ModelOptions:
+        if len(kwargs) > 0:
+            raise InvalidArgument("This model does not support any options.")
```

Comment:
More accurately this "framework" does not support any options. Can we pass the`info.module` field into `ModelOptions` and include the module name in the exception message?

Reply:
We probably could, let's do that.